src/applications/personalization/dashboard/containers/DashboardApp.jsx

```diff
@@ -196,7 +196,7 @@ class DashboardApp extends React.Component {
     const view = (
       <div className="row user-profile-row">
         <div className="usa-width-two-thirds medium-8 small-12 columns">
-          <h1>Your Homepage</h1>
+          <h1 id="dashboard-title">Your Homepage</h1>
           <div className="va-introtext">
```

Comment:
Wasn't happy about adding an `id`, but somehow we have style selectors like`.usa-width-one-whole:first-child > :first-child` that make things really difficult to override.